lib/shoulda/matchers/integrations/configuration.rb

```diff
@@ -0,0 +1,67 @@
+require 'set'
+
+module Shoulda
+  module Matchers
+    module Integrations
+      # @private
+      class Configuration
+        def self.apply(configuration, &block)
+          new(configuration, &block).apply
+        end
+
+        def initialize(configuration, &block)
```

Comment:
Unused method argument - `configuration`. If it's necessary, use `_` or `_configuration` as an argument name to indicate that it won't be used.